tests/TenantDatabaseManagerTest.php

```diff
@@ -225,7 +225,21 @@
 });
 
 test('path used by sqlite manager can be customized', function () {
-    pest()->markTestIncomplete();
+    Event::listen(TenantCreated::class, JobPipeline::make([CreateDatabase::class])->send(function (TenantCreated $event) {
+        return $event->tenant;
+    })->toListener());
+
+    // Set custom path for SQLite file
+    SQLiteDatabaseManager::$path = $customPath = storage_path();
```

Comment:
I wouldn't use `storage_path()` for everything, it makes the code unclear. It'd be better to either hardcode a path here or use some gitignored directory.

Reply:
So I used the `database_path('custom')` now. I think it makes sense because it's more likely developers put the sqlite database inside Laravel's `database` directory. Using this path works well with testbench means we don't need a gitignore directory etc.